CAP/examples/OppositeCategory.g

```diff
@@ -7,16 +7,20 @@ LoadPackage( "LinearAlgebraForCAP" );
 QQ := HomalgFieldOfRationals();;
 vec := MatrixCategory( QQ );;
 op := Opposite( vec );;
-ListKnownCategoricalProperties( op );
```

Comment:
Please keep the old version so we don't have to update even more numbers when adding or removing derivations. Also, this is only one special case. If you want to compare the numbers later on (as the commit message suggests) I would suggest adding an example file which covers all cases but is not executed as part of the tests.

Reply:
Done.